cmd/shortener/main_test.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"bytes"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func TestURLShortener_PostHandler(t *testing.T) {
+	us := &URLShortener{
+		urls:          map[string]string{},
+		ServerAddress: "localhost:8080",
+		BaseURL:       "http://localhost:8080/",
+	}
+
+	req, err := http.NewRequest(http.MethodPost, "http://localhost/", bytes.NewReader([]byte("https://www.example.com")))
+	assert.NoError(t, err)
+
+	w := httptest.NewRecorder()
+	us.PostHandler(w, req)
+	assert.Equal(t, http.StatusCreated, w.Code)
+
+	body := w.Body.String()
+	assert.Contains(t, body, "http://localhost")
```

Comment:
Было бы здорово добавить информативное сообщение в тесты о том, что ожидалось или что прошло не так. 
```suggestion
	assert.Contains(t, body, "http://localhost", "http://localhost should be in the response body")
```

Reply:
исправлено